client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/screen-main.tsx

```diff
@@ -0,0 +1,72 @@
+import { Button } from '@automattic/components';
+import {
+	__experimentalItemGroup as ItemGroup,
+	__experimentalDivider as Divider,
+} from '@wordpress/components';
+import { header, footer, layout } from '@wordpress/icons';
+import { useTranslate } from 'i18n-calypso';
+import { NavigationButtonAsItem } from './navigator-buttons';
+import NavigatorHeader from './navigator-header';
+
+interface Props {
+	onSelect: ( name: string ) => void;
+	onContinueClick: () => void;
+}
+
+const ScreenMain = ( { onSelect, onContinueClick }: Props ) => {
```

Comment:
The new layout of the sidebar
<img width="241" alt="image" src="https://user-images.githubusercontent.com/13596067/220594398-6003a499-9e9f-4108-a02f-a6b9c3d7577d.png">